src/Lykke.Service.Assets.Repositories/AssetConditionLayerRepository.cs

```diff
@@ -140,5 +129,25 @@ public static string GetAssetConditionRowKey(string asset)
             await _assetConditionLayerTable.DeleteAsync(GetAssetConditionLayerPartitionKey(),
                 GetAssetConditionLayerRowKey(layerId));
         }
+        
+        private static string GetAssetConditionLayerPartitionKey()
+        {
+            return "ConditionLayer";
```

Comment:
Use .CalculateHexHash32(n) of layerId to improve partitions segmentation

Reply:
This partition key is used in production environment. In this case it will be a small partition around 10-30 rows.